INFO     HTTP_CLIENT:event_hooks.py:24 Отправляем запрос: POST на http://localhost:8001/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:28 Получаем ответ: 200 OK от http://localhost:8001/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:24 Отправляем запрос: POST на http://localhost:8001/api/v1/authentication/login
INFO     HTTP_CLIENT:event_hooks.py:28 Получаем ответ: 200 OK от http://localhost:8001/api/v1/authentication/login
INFO     HTTP_CLIENT:event_hooks.py:24 Отправляем запрос: POST на http://localhost:8001/api/v1/exercises
INFO     HTTP_CLIENT:event_hooks.py:28 Получаем ответ: 422 Unprocessable Entity от http://localhost:8001/api/v1/exercises
INFO     BASE_ASSERTIONS:base.py:24 Проверяем, что статус код ответа сервера соответствует 422
INFO     EXERCISES_ASSERTIONS:exercises.py:188 Проверяем ответ сервера на запрос создания упражнения с некорректным id курса
INFO     ERRORS_ASSSERTIONS:errors.py:51 Проверяем ответ сервера с ожидаемой валидационной ошибкой
INFO     BASE_ASSERTIONS:base.py:82 Проверяем, что длина details соответствует ожидаемой [ValidationErrorSchema(type='uuid_parsing', input='incorrect_course_id', context={'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}, message='Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1', location=['body', 'courseId'])]
INFO     ERRORS_ASSSERTIONS:errors.py:28 Проверяем данные ожидаемой валидационной ошибки
INFO     BASE_ASSERTIONS:base.py:45 Проверяем, что значение type соответствует ожидаемому uuid_parsing
INFO     BASE_ASSERTIONS:base.py:45 Проверяем, что значение message соответствует ожидаемому Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1
INFO     BASE_ASSERTIONS:base.py:45 Проверяем, что значение location соответствует ожидаемому ['body', 'courseId']
INFO     BASE_ASSERTIONS:base.py:45 Проверяем, что значение input соответствует ожидаемому incorrect_course_id
INFO     BASE_ASSERTIONS:base.py:45 Проверяем, что значение context соответствует ожидаемому {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}
INFO     SCHEMA_ASSERTIONS:schema.py:26 Проверяем, соответствует ли ответ сервера JSON-схеме